notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<p>Please give a more precise definition than "performance": time per request halved ?</p><p>Because performance can also be "accuracy of the results", which will not be the case here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>50% *reduction in data retrieval time would be more accurate I think!</p>

Reply:
<p>Also, the title could be "Conclusion" without the plural and point at the end.</p>

Reply:
Done